apps/website/docs/react/guides/limitations.md

```diff
@@ -4,6 +4,52 @@ sidebar_position: 2
 
 # Limitations
 
+## Runtime styles
+
+Styles can't be created at runtime which includes dynamic selectors as well.
+
+```jsx
+function App(props) {
+  // ❌ This will not work and throw an expection
+  const useClasses = makeStyles({
+    color: props.color,
+    [`.${props.area}`]: { display: 'block' },
+  });
+}
+```
+
+As Griffel performs ahead-of-time [compilation](/react/ahead-of-time-compilation/introduction) values used in CSS rules should be static to be compiled.
+
+### Workarounds
+
+- Use inline styles on elements. They don't have the best performance, but it will be faster than invoke any CSS-in-JS for frequently changing values.
```

Comment:
```suggestion
- Use inline styles on elements. They don't have the best performance, but it will be faster than invoking any CSS-in-JS for frequently changing values.
```